lms/static/js/dashboard/dropdown.js

```diff
@@ -9,9 +9,15 @@ var edx = edx || {};
     // Generate the properties object to be passed along with business intelligence events.
     edx.dashboard.dropdown.toggleCourseActionsDropdownMenu = function(event) {
         // define variables for code legibility
-        var dashboardIndex = $(event.currentTarget).data().dashboardIndex,
-            $dropdown = $('#actions-dropdown-' + dashboardIndex),
-            dropdownButton = $('#actions-dropdown-link-' + dashboardIndex),
+        var $currentTarget = $(event.currentTarget),
+            dashboardIndex = $currentTarget.data().dashboardIndex,
+
+            dropdownId = $currentTarget.data('dropdownId') || '#actions-dropdown-' + dashboardIndex,
+            $dropdown = $(dropdownId),
+
+            dropdownButtonId = $currentTarget.data('dropdownButtonId') || '#actions-dropdown-link-' + dashboardIndex,
+            dropdownButton = $(dropdownButtonId),
```

Comment:
Nit, add a $ to the front of this (jquery variable rule), it will probably cause a quality issue

Reply:
I noticed this two, but it was originally written that way and I didn't want to go down the rabbit hole of refactoring the whole file. I'll try to fix it though.